test/modules/core/core/attribute.spec.js

```diff
@@ -36,6 +36,9 @@ test('WebGL#Attribute constructor/update/delete', t => {
   attribute.delete();
   t.ok(buffer._handle, 'External buffer is not deleted');
 
+  attribute = new Attribute(gl, {size: 1, isIndexed: true});
+  t.is(attribute.type, GL.UNSIGNED_INT, 'type is auto inferred');
+
   t.end();
```

Comment:
Add test case without `gl`, with message that emphasizes that this is a supported case?